DefaultWidget.ts

```diff
@@ -68,6 +68,28 @@ const MAPPING_EN: PerLanguageMapping = new Map([
 		`Paris is the <mask> of France.`,
 		`The goal of life is <mask>.`,
 	] ],
+	[ "zero-shot-classification", [
+		{
+			text: "I have a problem with my iphone that needs to be resolved asap!!",
+			labels: "urgent, not urgent, phone, tablet, computer",
+			multiclass: true,
+		},
+		{
+			text: "Last week I upgraded my iOS version and ever since then my phone has been overheating whenever I use your app.",
+			labels: "mobile, website, billing, account access",
+			multiclass: false,
+		},
+		{
+			text: "A new model offers an explanation for how the Galilean satellites formed around the solar system’s largest world. Konstantin Batygin did not set out to solve one of the solar system’s most puzzling mysteries when he went for a run up a hill in Nice, France. Dr. Batygin, a Caltech researcher, best known for his contributions to the search for the solar system’s missing “Planet Nine,” spotted a beer bottle. At a steep, 20 degree grade, he wondered why it wasn’t rolling down the hill. He realized there was a breeze at his back holding the bottle in place. Then he had a thought that would only pop into the mind of a theoretical astrophysicist: “Oh! This is how Europa formed.” Europa is one of Jupiter’s four large Galilean moons. And in a paper published Monday in the Astrophysical Journal, Dr. Batygin and a co-author, Alessandro Morbidelli, a planetary scientist at the Côte d’Azur Observatory in France, present a theory explaining how some moons form around gas giants like Jupiter and Saturn, suggesting that millimeter-sized grains of hail produced during the solar system’s formation became trapped around these massive worlds, taking shape one at a time into the potentially habitable moons we know today.",
+			labels: "space & cosmos, scientific discovery, microbiology, robots, archeology",
+			multiclass: true,
+		},
+		{
+			text: "Все счастливые семьи похожи друг на друга, каждая несчастливая семья несчастлива по-своему.",
+			labels: "English, Russian, Arabic, Urdu, Mongolian",
+			multiclass: false,
+		}
```

Comment:
I don't know if this would work.

The `MAPPING_EN` is only going to apply if the model's language is defined as `en`, I believe. On which model(s) do you intend this input to display?

Reply:
If it's for https://huggingface.co/joeddav/xlm-roberta-large-xnli (and similar models), maybe we need to create a `MAPPING_MULTILINGUAL`... what do you think @joeddav?

Reply:
This example is for english models, not multilingual. It's just showing that it can detect the language. For `joeddav/xlm-roberta-large-xnli` I think we can just use the model card.

That raises a question though: if the model doesn't have a lang tag, will it use `MAPPING_EN` by default or just not have a default?

Reply:
> This example is for english models, not multilingual. It's just showing that it can detect the language. For `joeddav/xlm-roberta-large-xnli` I think we can just use the model card.

Do all english models trained on NLI "know" how to identify languages?
In all cases i think it might be confusing for visitors of the website to see this in a widget for an EN model. Just my 2 cents though.

> That raises a question though: if the model doesn't have a lang tag, will it use MAPPING_EN by default or just not have a default?

If a model doesn't have a lang tag, it will currently use MAPPING_EN – but I'm not sure that's a good idea, might disable this in the future.

On the other hand, if a model has a lang tag for which we don't have sample inputs, there won't be any sample input.

Reply:
Anyways... feel free to merge when you're happy with this!

Reply:
Yeah, it can definitely identify languages. It can even do some limited crosslingual classification just because the pretraining dataset has some exposure to other languages, somewhat analogous to how GPT-2 can do translation despite their attempt to remove non-english text from the corpus perhaps. But anyway, I've removed it since it's more confusing than helpful.

> On the other hand, if a model has a lang tag for which we don't have sample inputs, there won't be any sample input.

Yeah this is the problem. There are too many untagged English models. I think this way is better.